src/RpcClient.ts

```diff
@@ -354,4 +382,111 @@ export class RpcClient {
       throw new Error(`Error in getSignaturesForAsset: ${error}`);
     }
   }
+  /**
+   * Mint a compressed NFT.
+   * @returns {Promise<MintCNFTResponse>}
+   * @throws {Error}
+   */
+  async mintCNFT(
+    request: MintCNFTRequest
+  ): Promise<MintCNFTResult | undefined> {
+    try {
+      const ownerKeypair = loadWallet(request.ownerKeypair);
+      function delay(ms: number) {
```

Comment:
don't need to define this function within the function, you can move it to a util and use it multiple times.